### PR TITLE
Remove scons support

### PR DESCRIPTION
**Pull request by bachase**
Removes scons support and switches to cmake for building rippled. The second changeset also removes several directories in `Builds` that are not actively maintained. The third changeset adds Ubuntu focused build instructions for linux that mirror the instructions for macos and VisualStudio2017.

### CODE REVIEW

**Comment by bachase** [comment]
I moved files like `Builds/Ubuntu/build_clang_libs.sh → Builds/linux/Ubuntu/build_clang_libs.sh`, but I'd prefer to remove them if they aren't really needed as part of this repository.

**Comment by seelabs** [comment]
@bachase It depends if we want to support clang on ubuntu distros earlier than 16.04. It's a strange enough combination at this point that I'd be OK not support that combo and removing the script.

File: Builds/linux/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+the `-j` parameter in this example tells the build tool to compile several
+files in parallel. This value should be chosen roughly based on the number of
+cores you have available and/or want to use for building.
+
+When the build completes succesfully, you will have a `rippled` executable in
+the current directory, which can be used to connect to the network (when
+properly configured) or to run unit tests.
+
+#### Options During Configuration:
```
Comment (MarkusTeufelberger) [inline Builds/linux/README.md]: There's also static and assertion enabled builds.

**Comment by bachase** [comment]
Rebased on 1.0.0-b2

**Comment by bachase** [comment]
Merged in 1.0.0-b3 f0b9506617416766c605fccfde051660076de16d

**Pull request closed without merging**